airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/testutils/NonContainer.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.testutils;
+
+import static io.airbyte.cdk.db.factory.DatabaseDriver.SNOWFLAKE;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+public class NonContainer extends JdbcDatabaseContainer<NonContainer> {
```

Comment:
Commentary required. Do I understand correctly that this is a shim for when a test relies on a shared instance somewhere rather than a testcontainer?